packages/core/src/2d/enums/SpriteTileMode.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Sprite's tiling mode enumeration.
+ * Only works in `SpriteDrawMode.tiled` mode.
+ */
+export enum SpriteTileMode {
+  /** SpriteRenderer will tile continuously. */
+  Continuous,
+  /** When the size of the intermediate expansion is greater than or equal to `SpriteRenderer._tileStretchValue`, 
+   * a tile will be added，and the number of tiles can only be an integer. */ 
+  Adaptive
+}
```

Comment:
/**
 * Tiling mode when the sprite drawing mode is tiled.
 */
export enum SpriteTileMode {
  /** Continuously tile sprites while rendering. */
  Continuous,
  
  /**
   * Adaptive tiling when rendering.
   * 
   * @remarks
   * When the size of the intermediate expansion is greater than or equal to `SpriteRenderer.tileStretchValue`, 
   * a tile will be added，and the number of tiles can only be an integer. 
   */
  Adaptive
}